tests/acceptance/features/bootstrap/WebUISharingContext.php

```diff
@@ -1973,4 +1973,20 @@ public function publicLinkSharesWithNameShouldBeVisibleOnTheWebUI($expectedCount
 			$expectedCount
 		);
 	}
+
+	/**
+	 * @When the user closes federation share dialog
+	 */
+	public function theUserClosesFederationShareDialog()
+	{
+		$this->filesPage->closeFederationDialog();
+	}
+
+	/**
+	 * @When the user accepts the pending remote shares using the webUI
```

Comment:
```suggestion
	 * @When the user accepts the pending remote share using the webUI
```
IMO the page object code will only accept 1 pending share (even if there are multiple waiting to be accepted). So we should write the singular in the step text, otherwise it will mislead people in the future who use this step.